robots/solo_description/robots/solo12.urdf

```diff
@@ -9,15 +9,15 @@
     <!-- BASE LINK INERTIAL -->
     <inertial>
       <origin rpy="0 0 0" xyz="0 0 0"/>
-      <mass value="1.63315091"/>
+      <mass value="1.16115091"/>
       <!-- The base is extremely symmetrical. -->
       <inertia ixx="0.00578574" ixy="0.0" ixz="0.0" iyy="0.01938108" iyz="0.0" izz="0.02476124"/>
     </inertial>
     <!-- BASE LINK VISUAL -->
     <visual>
       <origin rpy="0 0 0" xyz="0 0 0"/>
       <geometry>
-        <mesh filename="package://solo_description/meshes/obj/with_foot/solo_body.obj"/>
+        <mesh filename="package://example-robot-data/robots/solo_description/meshes/stl/solo12/solo_12_base.stl"/>
```

Comment:
A previous version of this PR had obj including material for the _visual_ meshes. They also appear to still be in the repository as the previous URDF had them. Does this PR change the look to uni-color due to this? I.e. a disimprovement in "looks"?

Reply:
For now, I have a uni-color robot. And according to https://github.com/open-dynamic-robot-initiative/robot_properties_solo/commit/266965a86759f99d196c6f5e4bf6cc61b038e272 I should see something better in gepetto-viewer. But I don't think it was working for me. I'll try again.

Reply:
@nim65s it seems you're using the STL for visualization. This kind of files doesn't contain color or texture information.

Reply:
It doesn't work. It looks like MTLs are for OBJs. If I add them both, and switch the urdf to obj, I load a bunch of cubes, and the robot inside. 

Plus anyway, the obj are not yet available for solo12, only for solo. If I generate those with assimp, I get the same result, and I don't think I'll get more info.

So, no, this new version of this PR is not a disimprovement in looks, as I don't think I ever see anything else than a uni-color robot.

Reply:
I'v pushed that on the https://github.com/Gepetto/example-robot-data/tree/topic/solo_color branch, if anyone wants to test, but I suggest to keep that as an issue for later.